slalexander92/react-challenge-project-jan-2023
# CHORE: Update Dependencies

## Changes
- update project dependencies to prevent security vulnerabilities
- update react + react libraries to most recent version(s)

## Purpose
- to close any known vulnerabilities due to out-of-date package dependencies
- update react to more modern tooling/syntax

## Approach
- close any known vulnerabilities due to out-of-date package dependencies, by updating packages

## Pre-Testing TODOs
N/A

## Testing Steps
- ensure functionality of app is intact (can login, order, view orders, log out, not hit pages that require authentication)

## Learning
- https://catalins.tech/update-project-dependencies-to-the-latest-versions/

Closes https://github.com/Shift3/react-challenge-project-jan-2023/issues/7
